boards/nxp/s32z2xxdc2/s32z2xxdc2_s32z270_rtu1_D.overlay

```diff
@@ -16,3 +16,19 @@
 	pinctrl-names = "default";
 	status = "okay";
 };
+
+/* Connect J32 pin 5 <-> J276 pin 3, J32 pin 6 <-> J276 pin 4 */
```

Comment:
So the EEPROM is present on the board, but not connected by default? If so, the devicetree node should be disabled by default.